### DISCUSSION TITLE
How to Prevent Or Remove noscript tag from gatsby-plugin-image ?

### DISCUSSION THREAD

**Discussion by willyw2k** [gatsbyjs/gatsby#30969] [Help]
How to Prevent Or Remove noscript tag from gatsby-plugin-image ?

<noscript><style>.gatsby-image-wrapper noscript [data-main-image]{opacity:1!important}.gatsby-image-wrapper [data-placeholder-image]{opacity:0!important}</style></noscript>

**Comment by juanbiberretta** [comment]
Yo! We had a similar issue. We do something like this on `gatsby-ssr.js`

```js
const flattenDeep = require('lodash/flattenDeep');

exports.onPreRenderHTML = ({ getHeadComponents, replaceHeadComponents }) => {
  const headComponents = flattenDeep(getHeadComponents());
  const filteredHeadComponents = headComponents.filter(
    (x) => x.key !== 'gatsby-image-style-noscript'
  );

  replaceHeadComponents(filteredHeadComponents);
};
```

**Reply by willyw2k** [reply]
Still The Same, Not Solved The Problem
